Cargo.toml

```diff
@@ -22,8 +22,7 @@ appveyor = { repository = "alexcrichton/futures-rs" }
 
 [features]
 use_std = []
```

Comment:
While we're at it I think the conventional name for this today is just `std`, so we could change that here too

Reply:
Done.